internal/exec/engine.go

```diff
@@ -272,7 +276,17 @@ func (e *Engine) fetchReferencedConfig(cfgRef types.ConfigReference) (types.Conf
 	if err != nil {
 		return types.Config{}, err
 	}
-	rawCfg, err := e.Fetcher.FetchToBuffer(*u, resource.FetchOptions{})
+	headers := make(http.Header)
```

Comment:
This means the headers specified only happen to config fetches, right

Reply:
Yeah.